src/App.js

```diff
@@ -1,13 +1,62 @@
 import React from "react";
 import "./App.css";
 
+
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+        score: 0,
+        increment: 1
+    }
+  }
+
+  incrementMethod = () => {
+    const {score, increment} = this.state; 
+    this.setState({
+        score: score + increment
+    })
+  }
+
+  scoreUpdateMethod = () => {
+    this.state.increment = this.state.increment + 1
+    this.setState({
+        score: this.state.score - 10
+    })
+  }
```

Comment:
```suggestion
  scoreUpdateMethod = () => {
  const { score, increment } = this.state //<--destructuring score and increment
    this.setState({
        score: score - 10,
        increment: increment + 1
    })
  }
```
State is being modified directly on line 22, which is **not good**. You should never ever ever modify state directly, **always use setState when you need to update state!**